content/kubermatic/v2.25/tutorials-howtos/kkp-autoscaler/_index.en.md

```diff
@@ -187,7 +187,16 @@ You can delete autoscaler from where you edit it above and select delete.
 ![Delete Autoscaler](/img/kubermatic/v2.25/tutorials/cluster-autoscaler/delete-autoscaler.png?classes=shadow,border "Delete Autoscaler")
 
 
- Once it has been deleted, you can check the cluster to ensure that the autoscaler has been deleted using the command `kubectl get pods -n kube-system`.
+ Once it has been deleted, you can check the cluster to ensure that the cluster autoscaler has been deleted using the command `kubectl get pods -n kube-system`.
+
+
+## Customize KKP Autoscaler
```

Comment:
Same as for the KubeOne document, I'd recommend using permalinks here.

Reply:
fixed!